apps/music/elements/music-overlay.js

```diff
@@ -165,20 +165,16 @@ proto.createdCallback = function() {
   this.els.message.dataset.l10nId = this.getAttribute('message-l10n-id');
 
   this.onDOMLocalized = () => {
```

Comment:
If you are renaming your event on document to "DOMRetranslated", please also rename this callback to onDOMRetranslated to match.